packages/vue/src/examples/pages/thank-you/ThankYou.vue

```diff
@@ -135,26 +205,47 @@ export default {
   }
 }
 .feedback-button {
-  margin: var(--spacer) 0;
+  margin: var(--spacer-sm) 0;
 
   @include for-desktop {
-    margin: 0 var(--spacer-medium) 0 0;
+    --button-width: 25rem;
```

Comment:
if all buttons on this page have 25rem you can use one place to set it width ;) wdyt